### PR TITLE
fix: display error message from chat conversation

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This PR adds capabilities to display error messages during a chat. So far we just printed the errors in the console, but they are useful to guide the user. 

I extended the messages that a conversation includes. So far, we just had ChatMessages, but we could also have ErrorMessages. The only thing i'm not sure about is how to display them, so i chose to use what we have now (just show the newest error message if it's the last message of the conversation), but it could be very easy to display them as part of the conversation so when you scroll up you can see the error you received for an old input. Waiting to hear your preference.

### Screenshot / video of UI

image

### What issues does this PR fix or reference?

it resolves https://github.com/containers/podman-desktop-extension-ai-lab/issues/1083 and https://github.com/containers/podman-desktop-extension-ai-lab/issues/1021

### How to test this PR?

1. open the playground
2. add a very long text ( > 2k chars), it should error (i used mistral model)

### CODE REVIEW

File: packages/shared/src/models/IPlaygroundMessage.ts [hunk 1]
```diff
@@ -18,11 +18,18 @@
...
 import type { ModelOptions } from './IModelOptions';
 
-export interface ChatMessage {
+export interface Message {
   id: string;
+  timestamp: number;
+}
+
+export interface ErrorMessage extends Message {
+  message: string;
```
Comment (axel7083) [inline packages/shared/src/models/IPlaygroundMessage.ts:27]: IMO this is not explanatory enough, might have something like `error: string` instead ?

Comment (lstocchi) [reply packages/shared/src/models/IPlaygroundMessage.ts]: updated 👍

**Review by axel7083 — APPROVED** [review]

**Review by jeffmaury — CHANGES_REQUESTED** [review]
Seems a unit test needs to be fixed

**Comment by lstocchi** [comment]
@jeffmaury are you good with this?

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**